src/app/forms/form-grid-validation/form-grid-validation.demo.html

```diff
@@ -21,11 +21,9 @@ <h3 class="modal-title">Modal</h3>
                     <div class="col-lg-4 col-md-4 col-sm-12 col-xs-12">
                         <input class="form-control" type="text" id="gForm_1" placeholder="Enter value here" size="45">
                     </div>
-                    <clr-tooltip
-                        [clrTooltipDirection]="'top-right'"
-                        [clrTooltipSize]="'lg'">
-                        <clr-icon shape="info-circle" class="info-tips-icon" size="24"></clr-icon>
-                        <clr-tooltip-content class="tooltip-content">
+                    <clr-tooltip>
+                        <clr-icon clrTooltipTrigger shape="info-circle" class="info-tips-icon" size="24"></clr-icon>
```

Comment:
This should be a link, shouldn't it? I think UX recommended actionable icons. I know the example earlier was just `clr-icon` but I can change it to a link if you confirm.